app/views/restaurants/index.html.erb

```diff
@@ -1,3 +1,39 @@
+<div class="container">
+
+    <div class="row">
+        <div class="col-lg-12 text-center">
+            <h2>Search Results: <span> NAME SEARCHED</span></h2>
+            <p class="lead"><span> 0 </span> results </p>
+            <button class="btn btn-default" role="button">Ratings</button>
+            <button class="btn btn-default" role="button">Price</button>
+        </div>
+    </div>
+
+</div>
+
+<hr>
+
 <% @restaurants.each do |restaurant| %>
-  <%= restaurant.inspect %>
+  <div class="col-sm-4 col-lg-4 col-md-4">
+    <div class="thumbnail">
+      <div class="result-details">
+        <h4><%= restaurant.name %></h4>
+        <%= link_to 'Show', restaurant %>
+        <p> District
+          <span class="pull-right">
+            <span> * </span>
+            <span> * </span>
+            <span> * </span>
```

Comment:
why do you have code that is very similar to `_header.html.erb`? can you extract these code into a partial?